evaluate.js

```diff
@@ -0,0 +1,85 @@
+Array.prototype.clean = function() {
+    for(let i = 0; i < this.length; i++) {
+        if(this[i] === "") {
+            this.splice(i, 1);
+        }
+    }
+    return this;
+}
+
+function expressionToRpn(expression) {
+        let outputQueue = "";
+        let operatorStack = [];
+        let operators = {
```

Comment:
it is a const, which won't be change
store it outside the function

Reply:
changed as recommended